### PR TITLE
Landing page avatar badge styleguide

### PR DESCRIPTION
**Pull request by palash23jain**

### CODE REVIEW

File: common-css/documentation.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.document-container {
+  display: grid;
+  grid-template-columns: 20% 80%;
+  font-size: var(--f5);
```
Comment (kiranpratyush) [inline common-css/documentation.css:4]: We can use better naming for variables

File: common-css/documentation.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  background-color: var(--secondary);
+}
+
+.component-list a.active-item {
+  color: var(--primary);
+  background-color: var(--text);
+}
+
+.document-main {
+  font-family: "Playfair", sans-serif;
```
Comment (kiranpratyush) [inline common-css/documentation.css:36]: I think font-family is inherited so we can set font-family in the body. We need not to redeclare it again

File: common-css/documentation.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  .sidebar {
+    display: none;
+  }
+  .hamburger-sidebar {
+    display: inline;
+  }
+  input[id="hamburger"] {
+    display: none;
+  }
+  input[id="hamburger"]:checked ~ .document-container {
```
Comment (kiranpratyush) [inline common-css/documentation.css:80]: We can use a separate class for the element

**Review by kiranpratyush — APPROVED** [review]
We can make the changes

File: common-css/style-guide.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  --text: #9c92a3;
+  --background: whitesmoke;
+  --heading: #221d23;
+  --gun-metal: #22333b;
+
+  /* typography */
+
+  /* font-size */
+
+  --f1: 0.8rem;
```
Comment (poornima-alugubelly) [inline common-css/style-guide.css:19]: This F1 S1 are confusing to read if you get time u can change them i guess

File: common-css/documentation.css (hunk 3 above)
Comment (palash23jain) [reply common-css/documentation.css:80]: Thanks @kiranpratyush for your feedback. Will work on them and try to make these improvements by next PR

**Review by poornima-alugubelly — APPROVED** [review]

File: common-css/style-guide.css (hunk 4 above)
Comment (palash23jain) [reply common-css/style-guide.css:19]: Thanks Poornima for your review. will make the changes by next PR.

**Pull request merged**